.vscode/ltex.hiddenFalsePositives.en-US.txt

```diff
@@ -0,0 +1,4 @@
+{"rule":"UPPERCASE_SENTENCE_START","sentence":"^\\Qjust-the-docs-template\\E$"}
+{"rule":"UPPERCASE_SENTENCE_START","sentence":"^\\Quses the Just the Docs theme\ncan be built and published on GitHub Pages\ncan be built and previewed locally (and published on other platforms).\\E$"}
+{"rule":"UPPERCASE_SENTENCE_START","sentence":"^\\Quses a gem-based approach, i.e. uses a \\E(?:Dummy|Ina|Jimmy-)[0-9]+\\Q and loads the \\E(?:Dummy|Ina|Jimmy-)[0-9]+\\Q gem;\nuses the GitHub Pages / Actions workflow to build and publish the site on GitHub Pages.\\E$"}
+{"rule":"UPPERCASE_SENTENCE_START","sentence":"^\\Quses a gem-based approach, i.e. uses a \\E(?:Dummy|Ina|Jimmy-)[0-9]+\\Q and loads the \\E(?:Dummy|Ina|Jimmy-)[0-9]+\\Q gem\nuses the GitHub Pages / Actions workflow to build and publish the site on GitHub Pages.\\E$"}
```

Comment:
I'm not familiar with this extension - what does this file do?

Reply:
This belongs to the previously mentioned "Ltex"-Extension. It was created when I added some of the issues in our readme.md as false positives. 
I committed that file to avoid a long list of warnings and errors in a repo the user hasn't even touched yet.

Reply:
A picture says more than a thousand words.

![image](https://github.com/just-the-docs/just-the-docs-template/assets/7556827/cff19c0a-2ca6-4ec2-9e48-61701e01d3ae)

Reply:
I disabled checking of our readme files and removed the falsePositives list.